app/helpers/vaccine_query_helper.rb

```diff
@@ -90,4 +90,9 @@ def paginate(vaccines, entries, page)
 
     vaccines.paginate(per_page: entries, page: page + 1)
   end
+
+  # Get vaccinations by patient_ids
+  def vaccines_by_patient_ids(patient_ids)
```

Comment:
minor thing, but can we move this to the top of the file for consistency (similar to `assessment_query_helper`)